daemon/src/projection.rs

```diff
@@ -1323,11 +1351,26 @@ impl CfdOffer {
             .iter()
             .map(|leverage| {
                 let liquidation_price = match own_position {
-                    Position::Long => calculate_long_liquidation_price(*leverage, offer.price),
-                    Position::Short => calculate_short_liquidation_price(*leverage, offer.price),
+                    Position::Long => calculate_long_liquidation_price(
+                        offer.price,
+                        offer.max_quantity,
+                        *leverage,
+                        offer.contract_symbol,
+                    ),
+                    Position::Short => calculate_short_liquidation_price(
+                        offer.price,
+                        offer.max_quantity,
+                        *leverage,
+                        offer.contract_symbol,
+                    ),
```

Comment:
❓ What should we do about this?

The liquidation price for ETHUSD is calculated based on the number of contracts, which is dynamic in the offer.

- Is the formula that we are using to calculate the quanto bankruptcy price correct?
- If so, is it okay to take an arbitrary value here since this is only informative?

Reply:
I don't think it would work well because:

- It only applies to ETHUSD. The value returned for BTCUSD will always be correct so it shouldn't be modified based on the number of contracts.
- The equation we have for ETHUSD is `liquidation_price = initial_price +/- (initial_margin / (multiplier * n_contracts))`. We would have to return all the components separately in order to compute it in the UI. It seems pretty complicated :(

Reply:
This sucks... Imho using `offer.max_quantity` here is wrong as it will show a very high (or low if going short) liquidation price. 

I guess we need to compute the whole liquidation price in frontend then

Reply:
It's interesting how the behaviour is completely different between inverse and quanto payout curves. Does this match your expectations? I've not gone back to verify the quanto formulas, but I did research it when I first implemented it.

In any case, I would prefer to document this bug in a separate issue and add it to the current milestone and iteration.

Reply:
Imho the formulas are correct if that is what you are asking. 
 
- either we decide to compute things in the daemon: then we need to add another http api which allows us to pass in the n_contracts the user wants and let the daemon compute those values
- or we compute these values purely in the frontend

I wanted to go for the second option when we ran into a similar problem with the inverse contracts. @da-kami convinced me otherwise and we made use of a nominal value and just _finalized_ it in the frontend. 

Happy to move on and create a bug/issue which captures this.

Reply:
I've created https://github.com/itchysats/itchysats/issues/2754.